scripts/check_frontend_test_coverage.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Check for decrease in coverage from 100% of frontend files."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import shutil
+import subprocess
+import sys
+
+import python_utils
+
+LCOV_FILE_PATH = '../karma_coverage_reports/lcov.info'
+DEV_LCOV_FILE_PATH = './tmp/dev-lcov.info'
+PR_LCOV_FILE_PATH = './tmp/pr-lcov.info'
+
+
+def change_git_branch(branch):
+    """Changes git branch.
+
+    Args:
+      branch: string. Name of the branch to be changed.
+    """
+    subprocess.check_call(['git', 'checkout', branch])
+
+    if branch == 'develop':
+        # This line is executed because CircleCI keeps the commits from PR
+        # branch when changing branchs. Ref ->
+        # (https://discuss.circleci.com/t/changing-git-branches-causes-
+        # commits-from-one-branch-to-carry-over-to-the-next/13562)
+        subprocess.check_call([
+            'git', 'reset', '--hard', 'origin/develop'])
+
+
+def run_frontend_tests_script():
+    """Run the frontend tests script using subprocess."""
+    subprocess.check_call([
+        'python', '-m', 'scripts.run_frontend_tests'])
+
+
+def create_tmp_folder():
+    """Creates a temporary folder."""
+    if not os.path.exists('./tmp'):
+        os.mkdir('./tmp')
+
+
+def delete_tmp_folder():
+    """Delete the temporary folder."""
+    if os.path.exists('./tmp'):
+        shutil.rmtree('./tmp')
+
+
+def filter_lines(line):
+    """Check if the line has file path (SF) or total lines (LF) or covered
+        lines (LH) of the test.
+
+    Args:
+        line: str. A line from lcov file.
+
+    Returns:
+        Boolean. If the line has the file path or total lines or covered lines
+        of the test.
+    """
+    return (line.find('SF') >= 0 or line.find('LH') >= 0
```

Comment:
You can probably do `return ('SF' in line or 'LH' in line or 'LF' in line)`

Might be shorter to write.

Reply:
Done